azurerm/resource_arm_automation_dsc_configuration_test.go

```diff
@@ -0,0 +1,130 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func TestAccAzureRMAutomationDscConfiguration_basic(t *testing.T) {
+	resourceName := "azurerm_automation_dsc_configuration.test"
+	ri := acctest.RandInt()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testCheckAzureRMAutomationDscConfigurationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAzureRMAutomationDscConfiguration_basic(ri, testLocation()),
+				Check: resource.ComposeTestCheckFunc(
+					testCheckAzureRMAutomationDscConfigurationExists(resourceName),
```

Comment:
We should be checking the values returned and read into state as can be see [here](https://github.com/terraform-providers/terraform-provider-azurerm/blob/master/azurerm/resource_arm_scheduler_job_test.go#L57)

Reply:
done